src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -38,7 +38,7 @@
       <NoWarn>NU1701</NoWarn>
     </PackageReference>
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.8100001-0126c21e" />
-    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.5.2-SNAPSHOT" />
```

Comment:
SNAPSHOT version and the actual version should exactly match. Please update the version to 1.5.3-SNAPSHOT

Reply:
There should not be any changes betwenn 1.5.2-SNAPSHOT and 1.5.2
If any changes are made on 1.5.2-SNAPSHOT, then you should release the next version.

Reply:
Done